massa-pool/src/operation_pool.rs

```diff
@@ -325,9 +325,11 @@ impl OperationPool {
                 .map(|op_id| {
                     self.ops
                         .get(op_id)
-                        .ok_or(PoolError::ContainerInconsistency(
-                            "op in ops by address is not in ops".to_string(),
-                        ))
+                        .ok_or_else(|| {
```

Comment:
sometimes it turns ok_or_else into ok_or, sometimes it does the opposite

Reply:
https://rust-lang.github.io/rust-clippy/master/#unnecessary_lazy_evaluations